### ISSUE TITLE
zip failed due to invalid structure

### ISSUE THREAD

**Issue by mjohsk** [kodi-pvr/pvr.iptvsimple#646]
I am trying to get iptv simple client to work in kodi nexus however I have no luck loading it as it fails with the error zip failed due to invalid structure.  Also internally trying to load pvr simple client without the latest zip file I will get the dependency on kodi binary instance 8.0.1 could not be satisfied.  I had all of it working recently but it must have updated and I cannot make it work since.  Thanks for your help

**Comment by phunkyfish** [comment]
Did you download the zip from GitHub? If so that won’t work. These are binary addons and must be compiled for your platform.

**Comment by mjohsk** [comment]
yes I did.   The internal pvr simple client within kodi gives the error - the dependency on kodi binary instance per version 8.0.1 could not be satisfied.  I have Uninstalled and reinstalled several different versions of nexus with the same results.   None of the downloads I can find work.  I was thrilled when it was working briefly as it was great.  Thanks for your help. --Sent from my Android phone with GMX Mail. Please excuse my brevity.On 9/20/22, 10:53 AM phunkyfish ***@***.***> wrote:

   
  Did you download the zip from GitHub? If so that won’t work. These are binary addons and must be compiled for your platform.

**Comment by phunkyfish** [comment]
What version of nexus are you using?

If you are using alpha3 you might need to switch to a nightly build as there was a PVR API change that likely means addons will not work with older versions.

Unfortunately the API change occurred after the Alpha3 release.

**Comment by mjohsk** [comment]
I have tried the alpha 3 release as well as most of the nightly builds and no luck so far

**Comment by phunkyfish** [comment]
Wait for the next mighty build for Nexus and it should work.

**Comment by mjohsk** [comment]
ok thanks thats what I am doing now as each one comes out
